src/Jebediah/Structure.hs

```diff
@@ -68,6 +76,6 @@ createLogStream group stream =
 
 newLogStream :: LogGroup -> LogStream -> AWS ()
 newLogStream g s = do
-  exists <- fmap (not . null) $ listLogStreams g (Just s)
+  exists <- fmap (not . null) $ listLogStreams g (StreamsPrefix $ Just s)
```

Comment:
Hmm. I think there may be an outstanding bug here. As won't be able to create 'a' if 'ab' exists?

Reply:
Yeah that seems likely. I'll put up a separate PR for that.